### ISSUE TITLE
Staging Review finding: Home address sections and edit buttons not easy to differentiate on review page

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#91905]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** IVC Forms
**Product:** 10-7959f-1 Foreign Medical Program (FMP) Registration
**Feature:** new product

## Findings details
**VA.gov Experience Standard - issue:** [User can't identify an element or its state.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

There are two pages in Step 4 that have the same H3, "Home address": [register-form-10-7959f-1/same-as-mailing-address](https://staging.va.gov/health-care/foreign-medical-program/register-form-10-7959f-1/same-as-mailing-address) and (if you answer "No") [register-form-10-7959f-1/home-address](https://staging.va.gov/health-care/foreign-medical-program/register-form-10-7959f-1/home-address).

Taken on their own, it's fine for these pages to have the same H3. But once you reach [register-form-10-7959f-1/review-and-submit](https://staging.va.gov/health-care/foreign-medical-program/register-form-10-7959f-1/review-and-submit), you have two sections in a row with the same heading level, the same heading text, and the same Edit button `aria-label` ("Edit home address").

For a screen reader user navigating by heading or tabbing through the Edit buttons, it's going to be difficult to distinguish these sections and land in the place you're expecting to land.

<details><summary>Link, screenshot or steps to recreate</summary>

![Part of the review page, with two successive headings with the same heading level and the same text, "Home address."](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/Brian-74187-1.png?raw=true)

</details>

## Recommended action

Adjust the H3s on these two pages to make them distinct -- eg. "Home address same as mailing" and "Home address" (CAIA folks may have feedback here). That should carry through to the review page heading structure and edit buttons.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by reiting** [comment]
<img width="758" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMCtvQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--58b7f1021874fc0e8c22f82c5ecdcdae950c5405/Screenshot%202024-09-03%20at%209.54.44%E2%80%AFAM.png" alt="Screenshot 2024-09-03 at 9.54.44 AM.png" />

**Comment by reiting** [comment]
@marywang2 this is merged and should be in staging.

**Comment by marywang2** [comment]
FYI @jamiefiore
